network.js

```diff
@@ -35,7 +35,10 @@ export async function getMasterSizeFromArtifact(GITHUB_TOKEN) {
       var zip = new AdmZip(res2.data);
       var zipEntries = zip.getEntries();
       for (let i = 0; i < zipEntries.length; i++) {
-        if (zipEntries[i].entryName === `apk-metric.json`) {
+        if (metricType === 'apk' && zipEntries[i].entryName === `apk-metric.json`) {
+          return JSON.parse(zip.readAsText(zipEntries[i]))[`master_size`];
+        }
+        if (metricType === 'bundle' && zipEntries[i].entryName === `bundle-metric.json`) {
```

Comment:
Don't keep two jsons . Add an entry in the schema. 
`{ 
master_size: --, 
bundle_size: --
}`
This will ensure there's always a single source of truth. Also,  the decision to fetch the latest metric can be kept independent. Once we have the details, then we decide which value to fetch.